source/get-started/bare-metal-install-desktop.rst

```diff
@@ -31,9 +31,7 @@ Preliminary steps
 
       <release-number> is the latest |CL| auto-numbered release.
 
-#. Verify and decompress the file per your OS.
-
-   * :ref:`download-verify-decompress`
+#. :ref:`verify-linux`.
```

Comment:
We should update this link to  `download-verify-decompress` doc; it should reference all OSes, not just Linux. 
I can rename this doc title, too, removing "decompress".